eg/examples/prqlc.md

```diff
@@ -0,0 +1,25 @@
+# prqlc
+
+Run the compiler interactively:
```

Comment:
can you change all these to start with lower case and not include the colon?

The colon and upper case is only in the `# Basic Usage` and other sections, which you don't have and aren't needed here.

Reply:
Alright